lib/static_sync/storage.rb

```diff
@@ -49,6 +49,7 @@ def storage
         :region                => @config.remote['region'],
         :aws_access_key_id     => @config.remote['username'],
         :aws_secret_access_key => @config.remote['password'],
+        :aws_session_token     => @config.remote['session_token'],
```

Comment:
What if this key doesn't exist? If we're specifying this, won't username/password also be not set?